docs/pages/application-access/okta/hosted-guide.mdx

```diff
@@ -90,6 +90,68 @@ Once the initial enrollment has completed, and if Teleport Identity
 is enabled, the enrollment workflow will guide you through enabling SCIM
 provisioning as well.
 
+## Configure Access List Synchronization
+
+If Teleport Identity is enabled, the enrollment workflow will also guide you to configure syncing of
+User Groups and Direct Application Assignments as access lists. This will ensure
+that permissions in Okta are modeled properly in Teleport, allowing Teleport users
+to see the same Okta applications that they would when viewing the Okta dashboard.
+
+### Configuring Default Owners
+
+![Set Default Owners](../../../img/enterprise/plugins/okta/okta-access-list-default-owners.png)
+
+Access lists must have owners, so the first step of setting up Okta Access List
+Synchronization is setting default owners for our new access lists. You can select
+any number of default owners and free-form enter the owners if they do not yet exist
+in the system. These owners can be later changed and will not be overwritten by the
+Okta Access List synchronization process.
+
+### Importing Okta User Groups as Access Lists
+
+![Import Okta User Groups](../../../img/enterprise/plugins/okta/okta-access-list-import-user-groups-all.png)
+
+After configurating default owners, you can customize which Okta User Groups should
+be imported. By default, all user groups will be imported. If you would like more
+control, you can untoggle the "Import All User Groups" toggler at the top right of
+the panel and configure multiple filters which will be matched against the group
+names.  For more information on filter syntax, refer to the Import Filters section
+below.
+
+<Admonition type="note">
+Only Okta User Groups with assignments will be imported as an Access List. If an Okta User
+Group has no assignments, it will not be imported until it has assignments.
+</Admonition>
+
+### Importing Okta Application Assignments as Access Lists
+
+![Import Okta Application Assignments](../../../img/enterprise/plugins/okta/okta-access-list-import-applications.png)
+
+In addition to importing Okta User Groups, you can also import direct application
+assignments within Okta as Access Lists as well. This behaves in exactly the same way
+as user groups.
+
+<Admonition type="note">
+Only Okta Applications with assignments will be imported as an Access List. If an Okta
+Application has no assignments, it will not be imported until it has assignments.
+</Admonition>
+
+### Import Filters
+
+Filters are supported as glob syntax and regex syntax. Glob syntax are strings with
+wildcards in them that represent arbitrary values where the wildcards go. For example,
+a glob that matches names with a prefix of "app" and a suffix of "salesforce" would
+look like `app*salesforce`.
+
+You can also use raw regular expressions so long as the filter string is prefixed with `^`
+and ends with `*`. This will support the [https://github.com/google/re2/wiki/Syntax](Google RE2) regular expression syntax.
```

Comment:
i think the syntax is backwards

<img width="506" alt="image" src="https://github.com/gravitational/teleport/assets/43280172/2e960c8d-40a1-4179-9f4e-f07d90d5c869">

from looking at other mdx, i think it's `[Google RE2](https://github.com/google/re2/wiki/Syntax)`

can you double check it on the site after editing?